pkg/metrics/metrics.go

```diff
@@ -83,15 +85,15 @@ var (
 		prometheus.CounterOpts{
 			Subsystem: constants.KueueName,
 			Name:      "admitted_workloads_total",
-			Help:      "Total number of admitted workloads per cluster_queue",
+			Help:      "Total number of admitted workloads per 'cluster_queue'",
 		}, []string{"cluster_queue"},
 	)
 
 	admissionWaitTime = prometheus.NewHistogramVec(
 		prometheus.HistogramOpts{
 			Subsystem: constants.KueueName,
 			Name:      "admission_wait_time_seconds",
-			Help:      "The wait time since a workload was created until it was admitted, per cluster_queue",
+			Help:      "The wait time since a Workload was created until it was admitted, per 'cluster_queue'",
```

Comment:
```suggestion
			Help:      "The time (seconds) between Workload `created` to `admitted` status, per 'cluster_queue'",
```
I'm assuming is seconds, but feel free to change it if needed.

Reply:
It's part of the metric name, so I'm not sure if we need to repeat that.

Also, `created` and `admitted` are not values or fields anywhere, so I don't think putting them in quotes is appropriate.